assets/base.css

```diff
@@ -553,6 +563,10 @@ table:not([class]) th {
   }
 }
 
+.left {
+  text-align: left;
+}
+
```

Comment:
Although this isn't required (`text-align: left;`) being the default value, it might be used for other elements and for instances where we'd need to override an existing `text-align` value.